toolkits/global/packages/global-popup/scss/50-components/_global-popup.scss

```diff
@@ -16,9 +16,9 @@
 	display: block;
 	top: $c-popup--close-top;
 	right: $c-popup--close-right;
-	height: 12px;
-	background: url("data:image/svg+xml;charset=utf-8,%3Csvg%20xmlns%3D'http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg'%20width%3D'12'%20height%3D'12'%20viewBox%3D'0%200%2012%2012'%3E%3Cpath%20fill%3D'%23666666'%20d%3D'M1.8%2012l-1.7-1.7%2010.2-10.3%201.7%201.7-10.2%2010.3zm10.1-1.7l-1.7%201.7-10.2-10.3%201.7-1.7%2010.2%2010.3z'%2F%3E%3C%2Fsvg%3E") no-repeat;
-	padding-right: 12px;
+	height: 16px;
+	background: url("data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjE2IiB2aWV3Qm94PSIwIDAgMTYgMTYiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48cGF0aCBkPSJtMi4yOTY3OTU3NSAxMi4yNzcyNDc4Yy0uMzk2NTg3NTcuMzk2NTg3Ni0uMzk0Mzg4NDcgMS4wMzI4MTA5LS4wMDA2MjE0OCAxLjQyNjU3NzkuMzk2NTEyMjcuMzk2NTEyMyAxLjAzMjQ2NzY4LjM5MzQ4ODggMS40MjY1Nzc5MS0uMDAwNjIxNGw0LjI3NzI0NzgyLTQuMjc3MjQ3ODcgNC4yNzcyNDc4IDQuMjc3MjQ3ODdjLjM5NjU4NzYuMzk2NTg3NSAxLjAzMjgxMDkuMzk0Mzg4NCAxLjQyNjU3NzkuMDAwNjIxNC4zOTY1MTIzLS4zOTY1MTIyLjM5MzQ4ODgtMS4wMzI0Njc3LS4wMDA2MjE0LTEuNDI2NTc3OWwtNC4yNzcyNDc4Ny00LjI3NzI0NzggNC4yNzcyNDc4Ny00LjI3NzI0NzgyYy4zOTY1ODc1LS4zOTY1ODc1Ny4zOTQzODg0LTEuMDMyODEwOTEuMDAwNjIxNC0xLjQyNjU3NzkxLS4zOTY1MTIyLS4zOTY1MTIyNi0xLjAzMjQ2NzctLjM5MzQ4ODc1LTEuNDI2NTc3OS4wMDA2MjE0OGwtNC4yNzcyNDc4IDQuMjc3MjQ3ODItNC4yNzcyNDc4Mi00LjI3NzI0NzgyYy0uMzk2NTg3NTctLjM5NjU4NzU3LTEuMDMyODEwOTEtLjM5NDM4ODQ3LTEuNDI2NTc3OTEtLjAwMDYyMTQ4LS4zOTY1MTIyNi4zOTY1MTIyNy0uMzkzNDg4NzUgMS4wMzI0Njc2OC4wMDA2MjE0OCAxLjQyNjU3NzkxbDQuMjc3MjQ3ODIgNC4yNzcyNDc4MnoiIGZpbGw9IiM2NjYiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==") no-repeat;
```

Comment:
I'm just wondering if the encoding is correct 🤔

Reply:
Yeah I've tested it and it works. Maybe it's because I base64 encoded it while the previous one wasn't base64?

Reply:
Good spot though. I've removed the base64 encoding, now it's just svg inline and it's less code overall which is good

Reply:
well, I remember that a long time ago we modified all the icons on the article pages to be `utf-8`but I don't remember exactly if it was for some incompatibility issue or for some performance tweaks. In any case, if it works I guess It's fine! I just wanted to double-check!

Reply:
oh, you've changed it! haha ok!